_data-prepper/pipelines/configuration/sinks/opensearch.md

```diff
@@ -50,45 +50,82 @@ pipeline:
+`number_of_shards` | No | Integer  | The number of primary shards that an index should have on the destination OpenSearch server. This parameter is effective only when `template_file` is either explicitly provided in Sink configuration or built-in. If this parameter is set, it would override the value in index template file. For more information, see [Create index]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/).
+`number_of_replicas` | No | Integer | The number of replica shards each primary shard should have on the destination OpenSearch server. For example, if you have 4 primary shards and set `number_of_replicas` to `3`, the index has 12 replica shards. This parameter is effective only when `template_file` is either explicitly provided in sink configuration or built-in. If this parameter is set, it would override the value in index template file. For more information, see [Create index]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/).
+`distribution_version` | No | String  | Indicates whether the sink backend version is Elasticsearch 6 or later. `es6` represents Elasticsearch 6. `default` represents the latest compatible backend version, such as Elasticsearch 7.x, OpenSearch 1.x, or OpenSearch 2.x. Default is `default`.
+`enable_request_compression` | No | Boolean | Whether to enable compression when sending requests to OpenSearch. When `distribution_version` is set to `es6`, default is `false`. For all other distribution versions, default is `true`.
+`action` | No | String | The OpenSearch bulk action to use for documents. Must be one of `create`, `index`, `update`, `upsert`, or `delete`. The default value is `index`.
+`actions` | No | List | A [list of actions](#actions) that can be used as an alternative to `action` that reads as a switch case statement which conditionally determines the bulk action to take for an event. 
+`flush_timeout` | No | Long | A long class that contains the duration in milliseconds to try packing a bulk request up to the `bulk_size` before flushing the request. If this timeout expires before a bulk request has reached the `bulk_size`, the request will be flushed. Set to `-1` to disable the flush timeout and instead flush whatever is present at the end of each batch. Default is `60,000` or one minute.
+`normalize_index` | No | Boolean | If true, the OpenSearch sink will try to make dynamic index names. Index names with format options specified in `${})` valid according to the [index naming restrictions]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/#index-naming-restrictions). Any invalid characters will be removed. Default value is false.
```

Comment:
I wasn't quite certain re: the meaning of the second sentence, so please confirm.

Reply:
Would "...specified inside the following characters...?" help?

Reply:
Yes, that would add more clarity, but I think we're missing a verb after the characters, so I inserted "are" but wasn't sure if that was the intended meaning.